### PR TITLE
Awesome books: Plain JavaScript with objects.

### PR DESCRIPTION
**Pull request by Jrmcuervo**
### Awesome Books
In this project we;

- Build a basic website that allows users to add/remove books from a list.
- Use JavaScript objects and arrays. 
- Dynamically modify the DOM and add basic events.

### CODE REVIEW

**Review by NahnahAJ — CHANGES_REQUESTED** [review]
Hi @Jrmcuervo and @ghreza-crypto , 

Great job working on your project. :fire: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
- [ ] Equal contribution from both partners :heavy_check_mark: 
- [ ] No linter errors :heavy_check_mark: 
- [ ] Good use of GitHub flow :heavy_check_mark: 
- [ ] Project works as expected :heavy_check_mark: 

## Required Changes ♻️
- The main changes are related to the 5th point of the [Professional-repo-rules](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md). Kindly fix those and you'd be off to your next project :100: 

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.MD [hunk 1]
```diff
@@ -14,164 +14,142 @@
...
+I would like to thank...
 
-> Give credit to everyone who inspired your codebase.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-I would like to thank microverse inc. for helping me in my professional carreer. 
+## 📝 License <a name="license"></a>
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+This project is [MIT](./LICENSE) licensed.
```
Comment (TadesseDev) [inline README.MD:153]: - [ ] Dear @Jrmcuervo, you did a great job making the changes. I notice the previous reviewer asked for solving the link broken issue with the MIT file. Though the link is working, I see you have noted removing the previous link, Please remove the one that is not working. 
- [ ] Also. you updated the link, but the MIT file you refer to is not currently updated to match your project details. Please make sure to update the `YEAR`, `PROJEC NAME`, and other details regarding your project. This is to make sure your project is professional.

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @Jrmcuervo 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.

199733072-bdacca19-8df9-43a7-be93-3aff2f1c15c8

 ## Highlights
 - [ ] Good job making most of the changes requested by the previous reviewer. 🙏

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [Optional] Please always go through your readme file and make sure everything is in place and looks professional. Remember recruiters are coming street to your project and how you document your readme says much about you. 
 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Felix45 — APPROVED** [review]
# Approved :green_circle: 
Hi @Jrmcuervo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights :heavy_check_mark: 
- [ ] No linter errors :white_check_mark: 
- [ ] Descriptive pull request title and summary :white_check_mark: 
- [ ] Repository has a professional readme file :white_check_mark: 
- [ ] GitHub flow is used correctly :white_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Shedrack-Sunday — COMMENTED** [review]
Hi  @Jrmcuervo 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

# Invalid Code Review Request 🔴 

You have submitted a project that does not have the current commit of your current partner. Kindly open a new PR for this change 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Pull request merged**